examples/enable/canvas_demo.py

```diff
@@ -1,27 +1,28 @@
-from enable.api import Canvas, Viewport, Window
+from traits.api import HasTraits, Instance
+from traitsui.api import Item, View
+
+from enable.api import Canvas, Viewport, ComponentEditor, Component
 from enable.tools.api import ViewportPanTool
-from enable.example_support import demo_main, DemoFrame
 
 
-class MyFrame(DemoFrame):
+class Demo(HasTraits):
+    canvas = Instance(Component)
 
-    def _create_window(self):
+    traits_view = View(Item('canvas', editor=ComponentEditor(),
+                            show_label=False, width=200, height=200),
+                       resizable=True, title="Canvas Example")
 
+    def _canvas_default(self):
         canvas = Canvas(bgcolor="lightsteelblue", draw_axes=True)
-        from basic_move import Box
+        from .basic_move import Box
```

Comment:
This demo doesn't seem to be working and I suspect that's because of this import - `examples` is not a python package and neither are any of it's sub dirs.